SCClassLibrary/scide_scqt/ScIDE.sc

```diff
@@ -24,12 +25,24 @@ ScIDE {
 	}
 
 	*handshake {
-		this.send(\classLibraryRecompiled);
-		this.send(\requestDocumentList);
-		this.send(\requestCurrentPath);
+		fork{
+			condvar = CondVar();
```

Comment:
Can we rename this from `condvar` to something descriptive?

Reply:
Note to self: warn if condvar is not nil

Reply:
I renamed condvar and added a nil check.